tests/path/general_path_generator_test.cpp

```diff
@@ -0,0 +1,99 @@
+#include <gtest/gtest.h>
+
+#include "kagen/context.h"
+#include "kagen/generators/path/path_directed.h"
+#include "util/utils.h"
+
+class PathGeneratorTestFixture : public ::testing::TestWithParam<kagen::SInt> {};
+INSTANTIATE_TEST_SUITE_P(PathGenerationTests, PathGeneratorTestFixture, ::testing::Values(10, 500, 16384, 30001));
+
+namespace {
+constexpr bool debug_output = false;
+auto find_start_vertex(const kagen::Graph& graph, kagen::SInt n) {
```

Comment:
keep KaGen function name style -> `FindStartVertex` etc (only for newly written code that is not c&p'd from other projects)?

Reply:
done